tsconfig.json

```diff
@@ -12,13 +13,13 @@
 		"noImplicitReturns": false,
 		"noUnusedParameters": true,
 		"noUncheckedIndexedAccess": true,
-		"resolveJsonModule": true,
+		"resolveJsonModule": false,
```

Comment:
Import assertions are required since node version 17.5.0.
This change aims to prohibit json imports?

Reply:
> Import assertions are required since node version 17.5.0.

The Import assertions (Import attributes?) feature seems still experimental, according to the [Node.js document](https://nodejs.org/api/esm.html#import-attributes).

> This change aims to prohibit json imports?

Yes.